webapp/src/mutator.ts

```diff
@@ -384,9 +385,17 @@ class Mutator {
         await undoManager.perform(
             async () => {
                 await octoClient.deleteBoardMember(member)
+                const user = await octoClient.getUser(member.userId)
+                if (user) {
+                    store.dispatch(removeBoardUsers([user]))
+                }
             },
             async () => {
                 await octoClient.createBoardMember(member)
+                const user = await octoClient.getUser(member.userId)
```

Comment:
Probably here we can save of doing this call receiving the user directly in the `deleteBoardMember` function, actually that remove the need of the two requests, and this is only done in one place in the code where you have the user in the context, so it is not hard to do.

Reply:
Done.